pkg/unidling/api/types.go

```diff
@@ -9,6 +9,10 @@ const (
 	// objects associated with the idled endpoints
 	UnidleTargetAnnotation = "idling.alpha.openshift.io/unidle-targets"
 
+	// PreviousScaleAnnotation contains the previous scale of a scalable object
+	// (currently only applied by the idler)
+	PreviousScaleAnnotation = "scale.openshift.io/previous-scale"
```

Comment:
Yeah, I would probably call this idling.alpha.openshift.io/previous-scale or something like that